### PR TITLE
158 | Meghna | error handling for gateway error response

### PR DESCRIPTION
**Pull request by MeghnaThoughtworks**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import './App.css';
 import React, {useEffect} from "react";
 import VerifyHealthId from './components/verifyHealthId/verifyHealthId';
 import './ndhm.scss';
+import OtpVerification from './components/otp-verification/otpVerification';
```
Comment (Mahesh-I) [inline src/App.js:5]: Can you remove coz we are not using OtpVerification here

File: src/components/verifyHealthId/verifyHealthId.jsx [hunk 2]
```diff
@@ -26,6 +35,7 @@ const VerifyHealthId = () => {
                         <input type="text" id="healthId" name="healthId" value={healthId} onChange={healthIdOnChangeHandler}/>
                     </div>
                     <button name="verify-btn" type="button" onClick={verifyHealthId} disabled={showAuthModes}>Verify</button>
+                    {showError && <div className="error">{errorHealthId}</div>}
```
Comment (Mahesh-I) [inline src/components/verifyHealthId/verifyHealthId.jsx:38]: Can we add the error message below to the input text box for all steps

Comment (MeghnaThoughtworks) [reply src/components/verifyHealthId/verifyHealthId.jsx]: it wont look good when the error comes below input field and shifts the button so having below button wont interfere with any position changes

File: src/App.js (hunk 1 above)
Comment (MeghnaThoughtworks) [reply src/App.js]: done

**Review by swatigogia2020 — APPROVED** [review]

**Review by Mahesh-I — APPROVED** [review]

**Pull request merged**